api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/TranslationProvider.java

```diff
@@ -165,7 +165,7 @@ private static String formatString(String textString, Object... parameters) {
 		if (parameters.length == 0) {
```

Comment:
While I was debugging I noticed that in some cases when {0} are used in the label translation, parameters.length is zero here, but later in I18nStringTemplateModel it is replaced as it should be (for instance for Taille minimale de l'image: {0} par {1} pixels). I hope @litvinovg is aware of this, and this is working in accordance with his expectations.